### ISSUE TITLE
 Does this include all the code

### ISSUE THREAD

**Issue by wyning1008** [HeLiancheng/HGRN#1]
I would like to reproduce some of the innovative points in the paper. Does this include all the code？

**Comment by HeLiancheng** [comment]
Yes, all the code is here. I'm sorry that, it's been a long time since I've logged on to GitHub
